tests/test_user/test_change_user_role.py

```diff
@@ -61,3 +64,13 @@ def test_change_user_role_with_already_assigned_role(self):
             change_user_role_with_already_assigned_role_mutation,
             change_user_role_with_already_assigned_role_mutation_response
         )
+
+        def test_restricting_admin_to_create_super_admin(self):
+            """
+            Testing restriction of an admin to create a super admin
+            """
```

Comment:
I don't understand the use of this block

Reply:
Kindly provide an answer to the above question